### ISSUE TITLE
On Singleplayer it says "You can not have a origin"

### ISSUE THREAD

**Issue by skxrlxt** [apace100/origins-fabric#523]
I've seen people with this issue but with them it happens on servers, mine is happening on singleplayer-- It didn't give me a prompt to pick origin and I cant even use the orb or commands because it comes up with this. 

I own an SMP where we use origins and we're updating to 1.18.2 soon and we're stumped as to why this is happening.
The jar file is labelled "Origins-1.18.2-1.4.1" if that helps.

![image](https://user-images.githubusercontent.com/87355517/163656858-1361d01e-406e-4410-b913-aa729c8ba8ef.png)

**Comment by skxrlxt** [comment]
UPDATE: There is an incompatibility with Charm, is there a way for this patched so we can have both?

**Comment by apace100** [comment]
Thanks for figuring out which mod causes the incompatibility. I'm currently out of town, so can't work on this, but I'll see what I can do when I'm back next week.

**Comment by Wahngrok** [comment]
I experienced the same bug on 1.18.2 today and was able to trace it to an incompatibility with DeepslateCutting (v1.3.2).

**Comment by philipwilk** [comment]
> UPDATE: There is an incompatibility with Charm, is there a way for this patched so we can have both?

Also experience this with Charm, disabled Charm, works perfectly.

**Comment by apace100** [comment]
Do any of you experiencing this bug have the mod OK Zoomer installed by any chance?

**Comment by Wahngrok** [comment]
> Do any of you experiencing this bug have the mod OK Zoomer installed by any chance?

I have both mods active and have had no problems.

**Comment by apace100** [comment]
> I have both mods active and have had no problems.

By both mods you mean OK Zoomer and DeepslateCutting? If you haven't had any problems, why have you reported experiencing this bug? Or did you mean Charm and OK Zoomer aren't causing problems for you, and they only happen once you had DeepslateCutting?

**Comment by Wahngrok** [comment]
> By both mods you mean OK Zoomer and DeepslateCutting?

No, I meant OK Zoomer and Origins. Sorry about not being clear. For me it is only Deepslate Cutting and Origins that are causing the error if both mods are active.

**Comment by apace100** [comment]
> > By both mods you mean OK Zoomer and DeepslateCutting?
> 
> No, I meant OK Zoomer and Origins. Sorry about not being clear. For me it is only Deepslate Cutting and Origins that are causing the error if both mods are active.

Even without OK Zoomer active?
My questions are:
Can you run Origins and DeepslateCutting alone?
Can you run Origins and OK Zoomer alone? You answered yes.
Can you run Origins and DeepslateCutting and OK Zoomer together?

Edit: I'm asking because this seems to be an OK Zoomer issue, and on my end I can't replicate the error with *just* OK Zoomer and Origins. But if you have an issue with just those two *without* OK Zoomer installed, then it's a separate issue.

**Comment by Wahngrok** [comment]
> Can you run Origins and DeepslateCutting alone?
> Can you run Origins and OK Zoomer alone? You answered yes.
> Can you run Origins and DeepslateCutting and OK Zoomer together?

Did some testing on a fresh instance with newly downloaded mods. Here are my results of testing:

- Origins + DeepslateCutting -> Origins selectable
- Origins + OK Zoomer -> Origins selectable
-  Origins + DeepslateCutting + OK Zoomer -> Origins **not** selectable

So it seems I did not test the first configuration before and must have thought the problem being with DeepslateCutting alone and not in combination with another mod. Sorry about that.

Mod versions used:
- deepslatecutting-1.18.2-1.3.2
- fabric-api-0.51.1+1.18.2
- okzoomer-5.0.0-beta.6+1.18.2
- Origins-1.18.2-1.4.1

Fabric Loader 0.13.3 (OK Zoomer causes crash on startup with Fabric Loader 0.14.0)

**Comment by Whocraf** [comment]
Having this problem with 1.20.1 and none of the aforementioned mods
